packages/core/src/lib/implementation/persist.integration.test.ts

```diff
@@ -56,29 +53,19 @@ const resetFiles = async () => {
   unlinkSync(reportPath('json'));
   unlinkSync(reportPath('md'));
 };
-const setupConsole = async () => {
-  logs = [];
-  mockConsole(msg => logs.push(msg));
-};
-const teardownConsole = async () => {
-  unmockConsole();
-};
 
 // @TODO refactor away from snapshots in favour of disc space and readability
 describe('persistReport', () => {
   beforeEach(async () => {
     resetFiles();
-    setupConsole();
   });
 
-  afterEach(() => {
-    teardownConsole();
-  });
+  afterEach(() => {});
```

Comment:
Can now be removed :slightly_smiling_face: 

```suggestion
```

Reply:
Oops 😄